src/components/bookings.tsx

```diff
@@ -1,9 +1,11 @@
 import classNames from 'classnames';
 import React, { useCallback, useContext } from 'react';
-import { AutoSizer } from 'react-virtualized/dist/es/AutoSizer';
-import { InfiniteLoader } from 'react-virtualized/dist/es/InfiniteLoader';
-import { List } from 'react-virtualized/dist/es/List';
-import { WindowScroller } from 'react-virtualized/dist/es/WindowScroller';
+import {
+  List,
+  WindowScroller,
+  InfiniteLoader,
+  AutoSizer,
+} from 'react-virtualized';
```

Comment:
The imports are separate to save bundle size. I just checked, merging them adds a whopping 12,5kb to the final bundle. Please remove this change from this PR.

Reply:
Why does webpack not automatically tree shake this?

Reply:
I think the default import is commonjs modules and these are harder to shake. Auto-treeshaking is still (2020) not fully reliable yet. This is also partly due to the nature of JS, where loading a module can have side-effects and thus webpack must be very conservative. ES-modules only partly have this problem (if all the module does is declare exports / classes / etc it's automatically pure and shakeable).

Reply:
As you know commonjs modules use assignment to a `module` field, and this cannot always be detected as _just that_ and nothing else (that would make the module impure to the outside world).